internal/featureflags/featureflags_test.go

```diff
@@ -0,0 +1,24 @@
+package featureflags
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestGet(t *testing.T) {
+
+	os.Setenv("FF_LOGS_V2", "true")
```

Comment:
My suggestion is to unset it at the end of the test so it doesn't cause unexpected behaviour in other tests